luigi/scheduler.py

```diff
@@ -1487,6 +1494,55 @@ def _first_task_display_name(self, worker):
         else:
             return task_id
 
+    @rpc_method()
+    def jobs(self):
+        try:
+            if configuration.get_config().get("kubernetes", "auth_method") == "kubeconfig":
+                config.load_kube_config()
+            else:
+                config.load_incluster_config()
+
+            batch_api_v1 = client.BatchV1beta1Api()
+
+            namespace = configuration.get_config().get("kubernetes", "namespace", "default")
+
+            result = []
+            for cron_job in batch_api_v1.list_namespaced_cron_job(namespace).items:
+                last_run = cron_job.status.last_schedule_time
+                result.append({
+                    "name": cron_job.metadata.name,
+                    "last_scheduled_run_at": last_run.strftime("%Y-%m-%d %H:%M:%S") if last_run else "Never ran"
+                })
+
+            return result
+        except (TypeError, NameError):
+            logger.error("Missing Kubernetes configurations")
+
+    @rpc_method()
+    def trigger_job(self, job_name):
+        try:
+            if configuration.get_config().get("kubernetes", "auth_method") == "kubeconfig":
+                config.load_kube_config()
+            else:
+                config.load_incluster_config()
+
+            batch_api_v1 = client.BatchV1beta1Api()
+
+            namespace = configuration.get_config().get("kubernetes", "namespace", "default")
```

Comment:
I think it's better to make the namespace option as more dynamic instead of only using `default` namespace. However we can still put the `default` as the default value.

Reply:
@irvifa the current version reads the `namespace` value from the Luigi configuration https://luigi.readthedocs.io/en/stable/configuration.html#kubernetes. If the `namespace` configration is not set, it uses the `default` namespace. Do you suggest that the namespaces could be displayed in the page and one can choose a namespace and list the jobs there?

Reply:
Sorry for not making myself clear enough, yes, that's what I mean..